languages/python/django-oso/django_oso/partial.py

```diff
@@ -76,11 +79,10 @@ def isa_expr(expr: Expression, model: Model, **kwargs):
         model = getattr(model, attr).field.related_model
     constraint_type = apps.get_model(django_model_name(right.tag))
     assert not right.fields, "Unexpected fields in matches expression"
-    return None if issubclass(model, constraint_type) else FALSE_FILTER
+    return TRUE_FILTER if issubclass(model, constraint_type) else FALSE_FILTER
```

Comment:
I know this makes this code slightly cleaner (can remove `None` checks in a few spots -- although I don't see that actually in this PR), but it pollutes the stringified output for filters, making the tests harder to read and understand.

Should we reconsider just keeping this as `None`?

Reply:
We would both prefer to leave it like this. It's nice that all translation components now _always_ return filters. Using "always true" and "always false" representations that Django's ORM knows how to understand means we can lean on its boolean logic handling instead of duplicating that in our own code.

Reply:
FWIW, we tried using `None` both last night + again just now to see if it's possible, and it quickly became more complicated than this implementation. Agree that the filters look a little funky, but all of the funkiness is optimized away by Django's ORM so that the funkiness won't be visible to end users.

Reply:
Removed `None` checks: 7dfa0be